src/components/dashboard/DashboardDetail.js

```diff
@@ -116,15 +118,16 @@ class DashboardDetail extends Component {
          newcontacts
       } = this.state
 
-      const { authorizedApps, amountStr } = this.props
+      const { authorizedApps, amount } = this.props
 
       return (
          <PageContainer
             title={(
                <Fragment>
                   <span className='balance'>Balance: </span>
-                  <span className='color-black'>{amountStr}</span>
-                  <span className='near'>Ⓝ</span>
+                  {amount ?  <Balance
```

Comment:
nit: better to format in a way when it's clear what is which branch of ternary, i.e. something like:

```
{amount
    ? <Balance. amount={amount} milli={milli} />
    : "NaN"}
```

Reply:
(this applies to all uses of ternary operator, it becomes hard to read when also mixed with JSX)

Reply:
changed